### DISCUSSION TITLE
Servo

### DISCUSSION THREAD

**Discussion by kotovasia5120** [kachurovskiy/nanoels#202] [Q&A]
I realized that the price difference between the stepper motor kit and the servo kit is small compared to the advantages.
How hard will it be to use them in the program?

[Servo](https://sl.aliexpress.ru/p?key=WMWNs4o)

**Comment by kachurovskiy** [comment]
Please check the driver datasheet to ensure it can be controlled with 5V digital pulse and direction signals. Never used them myself.
